### ISSUE TITLE
Specify Mood Directory

### ISSUE THREAD

**Issue by jdrox** [clementine-player/Clementine#5950]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System:  Windows 7 x64
- Clementine version:  1.3.1

Please, allow us to keep .mood files in a directory of our choosing.  This would make manual file organization easier (e.g. the ability to drag and drop artist folders, in Windows, from a PC to a portable device, without copying over all of the unneeded .mood files, since the device doesn't support Clementine and therefore would just be wasting space and transfer time.

Thank you.

**Comment by Moilleadoir** [comment]
If you look in _Preferences | User Interface | Appearance_ you should find a checkbox at the bottom:
_Save .mood files in your music library_

When unchecked .mood files will be saved in a config folder somewhere, not with the music files.

**Comment by jdrox** [comment]
No, when the box is unchecked, the .mood files are not saved anywhere on disk.
